main.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/josephburnett/jd/web/serve"
 )
 
-const version = "1.7.1"
+const version = "1.8.0"
```

Comment:
Please remove this change. And make your merge request target `master`. All new changes land there first. Then I'll cut a new release branch and build new binaries. You have the number right; I'll release version `1.8.0` but I won't backport this feature to the 0.7 release branch. That's just for critical bug fixes. Thanks!!